### ISSUE TITLE
Has Not Connected being considered a good connection state

### ISSUE THREAD

**Issue by SinZ163** [GetDotaStats/site#249]
https://github.com/GetDotaStats/site/blob/master/site_files/global_functions.php#L1812

http://getdotastats.com/#s2__match?id=4328085

**Comment by jimmydorry** [comment]
Correct. If you are the cause of not loading, not connected becomes a failure. States that are bad would be something we could punish for, so I don't believe not connected should count. Especially since other kinds of failure could cause this (like the host starting a game, then immediately closing the game).

**Comment by jimmydorry** [comment]
It would make sense as orange though, so lets do that.
